graphics/webp-pixbuf-loader/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
+
+github.setup        aruiz webp-pixbuf-loader 0.2.7
+revision            0
+categories          graphics
+maintainers         nomaintainer
+license             lgpl-2
+description         WebP pixbuf loader for GDK Pixbuf
+long_description    A GdkPixbuf loader that enables WebP image support within GTK applications.
+
+master_sites        https://github.com/${github.author}/${github.project}/archive/refs/tags
+distname            ${version}
+worksrcdir          ${name}-${version}
```

Comment:
none of this should be needed; please try again after adding the `github.tarball_from` line